"""corrected attribute typo error

Revision ID: fa1ffa26f53a
Revises: 033996d743d4
Create Date: 2025-01-16 14:29:41.448971

"""
from typing import Sequence, Union

from alembic import op
import sqlalchemy as sa


# revision identifiers, used by Alembic.
revision: str = 'fa1ffa26f53a'
down_revision: Union[str, None] = '033996d743d4'
branch_labels: Union[str, Sequence[str], None] = None
depends_on: Union[str, Sequence[str], None] = None


def upgrade() -> None:
    # ### commands auto generated by Alembic - please adjust! ###
    op.add_column('auctions', sa.Column('users_id', sa.UUID(), nullable=True), schema='auctora_dev')
    op.drop_index('ix_auctora_dev_auctions_user_id', table_name='auctions', schema='auctora_dev')
    op.create_index(op.f('ix_auctora_dev_auctions_users_id'), 'auctions', ['users_id'], unique=False, schema='auctora_dev')
    op.drop_constraint('auctions_user_id_fkey', 'auctions', schema='auctora_dev', type_='foreignkey')
    op.create_foreign_key(None, 'auctions', 'users', ['users_id'], ['id'], source_schema='auctora_dev', referent_schema='auctora_dev')
    op.drop_column('auctions', 'user_id', schema='auctora_dev')
    # ### end Alembic commands ###


def downgrade() -> None:
    # ### commands auto generated by Alembic - please adjust! ###
    op.add_column('auctions', sa.Column('user_id', sa.UUID(), autoincrement=False, nullable=True), schema='auctora_dev')
    op.drop_constraint(None, 'auctions', schema='auctora_dev', type_='foreignkey')
    op.create_foreign_key('auctions_user_id_fkey', 'auctions', 'users', ['user_id'], ['id'], source_schema='auctora_dev', referent_schema='auctora_dev')
    op.drop_index(op.f('ix_auctora_dev_auctions_users_id'), table_name='auctions', schema='auctora_dev')
    op.create_index('ix_auctora_dev_auctions_user_id', 'auctions', ['user_id'], unique=False, schema='auctora_dev')
    op.drop_column('auctions', 'users_id', schema='auctora_dev')
    # ### end Alembic commands ###
